core/src/main/java/org/elasticsearch/env/NodeEnvironment.java

```diff
@@ -81,9 +81,6 @@
  * A component that holds all data paths for a single node.
  */
 public final class NodeEnvironment  implements Closeable {
-
-    private final ESLogger logger;
```

Comment:
I moved these down below the constants. I thought I was going to add more members so I wanted to fix the order. I didn't end up adding more members but I still like having the members in a sensible order.